### PR TITLE
Feature a

### PR DESCRIPTION
**Pull request by kumar88sunil**

### CODE REVIEW

File: fil6 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+kjajdaljdka
```
Comment (vistasunil) [inline fil6:1]: sort this out

File: fil6 [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+akjdhdkjk
+kjdakdjkj
+djsjadjsjda
+sdnsdadka
```
Comment (vistasunil) [inline fil6:5]: need rework

**Review by vistasunil — CHANGES_REQUESTED** [review]

File: fil6 (hunk 1 above)
Comment (kumar88sunil) [reply fil6:1]: fixed.

File: fil6 (hunk 2 above)
Comment (kumar88sunil) [reply fil6:5]: done

**Review by vistasunil — APPROVED** [review]

**Pull request merged**